src/MasterSite.php

```diff
@@ -195,6 +195,16 @@ function ( $block_content, $block ) {
 		add_action( 'admin_notices', [ $this, 'show_dashboard_notice' ] );
 		add_action( 'wp_ajax_dismiss_dashboard_notice', [ $this, 'dismiss_dashboard_notice' ] );
 		add_filter( 'timber/twig', [ $this, 'p4_optimize_img_url' ] );
+
+		// Pin the ElasticPress to v3.4 search algorithm.
+		if ( \is_plugin_active( 'elasticpress/elasticpress.php' ) ) {
+			add_filter(
+				'ep_search_algorithm_version',
+				function() {
+					return '3.4';
```

Comment:
We have a wrapper function around `add_filter` for in case the function only returns a static value, without needing to write the function.

```
simple_value_filter( 'ep_search_algorithm_version', '3.4');
```